docs/src/getting_started/cargo_install.md

```diff
@@ -1 +1,43 @@
-# CLI installation with cargo
+# CLI Installation with Cargo
+
+## Prerequisites
+The best way to build Arbiter today is from source. 
```

Comment:
I would actually say that it's much easier to install from crates with `cargo install arbiter` than from the source. I think it's good to include instructions for both but i crates is easier because you don't have to build from src.

Reply:
good point. will rework this part and probably just nix this particular line